geomstats/geometry/hyperbolic.py

```diff
@@ -83,41 +95,48 @@ def __init__(self, dimension, point_type='extrinsic', scale=1):
 
     @staticmethod
     def _belongs_ball(point, tolerance=TOLERANCE):
-        """Evaluate if a point belongs to the Hyperbolic space (poin. ball).
+        """Test if a point belongs to the hyperbolic space.
 
-        Evaluate if a point belongs to the Hyperbolic space based on
+        Test if a point belongs to the hyperbolic space based on
         the poincare ball representation, i.e. evaluate if its
-        squared norm is lower than one.
+        squared norm is lower than 1.
 
         Parameters
         ----------
         point : array-like, shape=[n_samples, dimension]
-                Input points.
+            Point.
```

Comment:
"Point on the manifold."?

Reply:
I wrote "Point to be tested." because we do not know if it is on the manifold.